libflux/flux-core/src/semantic/doc.rs

```diff
@@ -170,23 +190,23 @@ fn parse_package_values(
     let mut members: BTreeMap<String, Doc> = BTreeMap::new();
     let mut diagnostics: Diagnostics = Vec::new();
     for stmt in &f.body {
-        if let Some((name, comment, loc)) = match stmt {
+        if let Some((name, comment, loc, is_option)) = match stmt {
```

Comment:
Can we change `is_option` into an enum? The enum can have an `is_option()` function attached to it that returns a boolean, but I'd like it if this was typed.